kyuubi-common/pom.xml

```diff
@@ -59,6 +59,11 @@
             <artifactId>hadoop-common</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
do we also need to exclude common2/3 from  other deps?

Reply:
I'm not sure `hadoop-common` works without `common2`